ironfish/src/rpc/routes/wallet/burnAsset.ts

```diff
@@ -60,10 +62,15 @@ router.register<typeof BurnAssetRequestSchema, BurnAssetResponse>(
       throw new ValidationError('Invalid burn amount')
     }
 
+    const assetBuffer = Buffer.from(request.data.assetId, 'hex')
+
+    const assetResponse = await node.chain.getAssetById(assetBuffer)
```

Comment:
you can just call this `asset` its not a response
```suggestion
    const asset = await node.chain.getAssetById(assetBuffer)
```